### ISSUE TITLE
docker logs support

### ISSUE THREAD

**Issue by axi43** [zuazo/dockerspec#3]
Need support to be able to view docker logs stdout of the described container to test expected content.
Currently to do that we needed to run the command in a shell and store in a variable, but prefer to have out of the box framework support.

Below is the workaround example:

```ruby
logs = `docker logs #{described_container} 2>&1`
expect(logs).to include('some text here')
```

**Comment by zuazo** [comment]
Implemented in master through 0a2462ab9fd923bd45288785df9d29e725c81ce4.

**Comment by zuazo** [comment]
Released in [`0.4.0`](https://github.com/zuazo/dockerspec/releases/tag/0.4.0).
